### PR TITLE
Update get-started-core-ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Update per user licensing as well

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8813

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by v-tophillips — APPROVED** [review]

File: microsoft-365/compliance/get-started-core-ediscovery.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ Licensing for eDiscovery (Standard) requires the appropriate organization subscr
 
 - **Per-user licensing:** To place an eDiscovery hold on mailboxes and sites, users must be assigned one of the following licenses, depending on your organization subscription:
 
+  -  Exchange online Plan 2 license
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/get-started-core-ediscovery.md:39]:
```suggestion
  
  -  Exchange Online Plan 2 license
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**